drivers/dma/dma_intel_adsp_gpdma.c

```diff
@@ -219,6 +219,24 @@ static void intel_adsp_gpdma_select_owner(const struct device *dev)
 #endif /* CONFIG_DMA_INTEL_ADSP_GPDMA_NEED_CONTROLLER_OWNERSHIP */
 }
 
+#ifdef CONFIG_PM_DEVICE
```

Comment:
PM only?

Reply:
yep, that is weird.  Why `intel_adsp_gpdma_claim_ownership` is not unconditionally built while `intel_adsp_gpdma_release_ownership` is only built when `CONFIG_PM_DEVICE` is defined ?

Reply:
If PM_DEVICE is not enabled there is no use of this function.